### PR TITLE
Convert man pages to Markdown

### PR DESCRIPTION
**Pull request by jsquyres**
Inspired by https://github.com/jsquyres/pmix-master/pull/new/pr/pandoc-generated-man-pages, use Pandoc to generate nroff man pages, instead of maintaining nroff files manually.

### CODE REVIEW

**Comment by jsquyres** [comment]
Note that this PR will likely fail CI until everyone can update their setups to have Pandoc >=v1.12 available.

**Comment by jsquyres** [comment]
@rhc54 I don't know who maintains the various CI systems that failed -- can you ping them to install/update Pandoc >=v1.12?

Fair waiting: PRRTE is next.  😄

**Comment by rhc54** [comment]
@jjhursey @karasevb @artpol84 Can you please update your CI systems:

> install/update Pandoc >=v1.12

Thanks @jsquyres !

**Comment by rhc54** [comment]
@jsquyres PRRTE uses the same CI machines, so once we get them updated, both repos should be fine.

**Comment by jjhursey** [comment]
I updated the PMIx/PRRTE images when I updated OMPI. So the IBM run tests should be fine.
I'll take a look at the cross-version image to see if that is the reason why it is failing.

**Comment by jjhursey** [comment]
bot:ibm:xver:retest
bot:ibm:xvertool:retest
bot:ibm:xvercheck:retest

**Comment by jjhursey** [comment]
IBM CI looks good now. The cross version check image just needed pandoc it seems.

**Comment by jsquyres** [comment]
bot:mellanox:retest

**Comment by jsquyres** [comment]
@karasevb @artpol84 @jladd-mlnx @artemry-mlnx Can someone please add Pandoc >=v1.12 to the Mellanox CI for PMIx and PRRTE?  Thanks!

**Comment by jjhursey** [comment]
bot:ibm:xvercheck:retest

**Comment by artpol84** [comment]
@artemry-mlnx can you take a look on https://github.com/openpmix/openpmix/pull/1752#issuecomment-623021494?

**Comment by artemry-mlnx** [comment]
@jsquyres could you please add 'pandoc' here within this PR?https://github.com/openpmix/openpmix/blob/8c457aa469d87b76477da74e66622bda6d066b9e/.github/actions/mlnx/Dockerfile#L4

**Comment by jsquyres** [comment]
> @jsquyres could you please add 'pandoc' here within this PR?

Ah!  I didn't realize that the Dockerfile was here in this repo.  Thanks!

**Comment by jsquyres** [comment]
@artemry-mlnx Apparently there's no "pandoc" in the default Centos 7 yum repo:

```
2020-05-10T12:07:23.2503376Z Step 2/4 : RUN yum install -y perl perl-Data-Dumper     automake libtool flex make bzip2 git which rpm-build libevent-devel pandoc
2020-05-10T12:07:26.6389067Z  ---> Running in e4964817522c
2020-05-10T12:07:27.1209091Z Loaded plugins: fastestmirror, ovl
2020-05-10T12:07:27.4306710Z Determining fastest mirrors
2020-05-10T12:07:30.7888043Z  * base: centos.s.uw.edu
2020-05-10T12:07:30.7888744Z  * extras: mirror.shastacoe.net
2020-05-10T12:07:30.7890783Z  * updates: mirror.shastacoe.net
2020-05-10T12:07:32.9555095Z No package pandoc available.
```

File: .github/actions/mlnx/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM centos:7.6.1810
```
Comment (artemry-mlnx) [inline .github/actions/mlnx/Dockerfile:2]: Please add `RUN yum install -y epel-release` here.

Comment (jsquyres) [reply .github/actions/mlnx/Dockerfile:2]: That got it.  Thanks!

**Comment by rhc54** [comment]
@jjhursey I suspect the cross-version of `make check` failed for this PR against itself because @jsquyres renamed the wrapper compiler to `pmixcc`, and so none of the tests actually built. I'll investigate this separately.

**Comment by jsquyres** [comment]
bot:ibm:retest

**Comment by jsquyres** [comment]
> @jjhursey I suspect the cross-version of `make check` failed for this PR against itself because @jsquyres renamed the wrapper compiler to `pmixcc`, and so none of the tests actually built. I'll investigate this separately.

I don't think I renamed the PMIx wrapper to be `pmixcc` - I think it was already named that...?

**Comment by rhc54** [comment]
My bad - you had made a comment about it in a note you sent me, and somehow I translated that into a proposal to change the name. I honestly never use that wrapper compiler and totally forgot what I named it way back when 😄

**Pull request merged**